### DISCUSSION TITLE
Kaum noch Daten vom WR - wie Ursache finden/eingrenzen?

### DISCUSSION THREAD

**Discussion by eastman-xy** [tbnobody/OpenDTU#1472] [Q&A] [answered]
Hallo,
seit ca. 5 Wochen werden die Daten von meiner OpenDTU immer weniger. Mal nach 30min, manchmal erst wieder ein paar Stunden später. Hat jemand eine Idee, wie ich die Ursache finden kann? Kann ich, z.B. mittels Notebook/Handy) prüfen, ob der WR (HM-600) noch regelmäßig Datenpakete sendet? Die Konsole der OpenDTU (Olimex ESP32-POE-ISO + EBYTE nRF24L01) sagt, dass keine Pakete empfangen werden, weshalb ich die Ursache eher beim WR vermute. Eine Positionsänderung und/oder Sendeleistungänderung der OpenDTU hat nicht verändert.

**Comment by brainbarry** [comment]
ob es sein könnte dass die zunehmenden Wolken zu Abbrüchen führen?
Klären könnte das vielleicht eine Überwachung mit Grafana über Prometheus.
Das Projekt ist auch hier auf github zu finden.

**Reply by eastman-xy** [reply, marked as answer]
Ursache wurde gefunden. Ein zusätzlicher Kondensator (100µF) an der Spannungsversorgung des Funkmoduls hat für Abhilfe gesorgt.
Jetzt kommen die Daten im Sekundentakt.
